### DISCUSSION TITLE
Is this a bug?

### DISCUSSION THREAD

**Discussion by apprat** [prisma/prisma#11259] [Q&A]
Prisma shared connection, transaction affects other request
![80NS`~DPOZ`AO)C}1VWE2HT](https://user-images.githubusercontent.com/42712641/150069123-85445202-3304-4fc1-8208-736e8bad67bf.png)
s

**Comment by austincrim** [comment]
Hey @apprat 👋 ,

I'm unsure what problem you are running into.

1. Would you mind sharing what results you are expecting vs. what is actually happening? 
1. Is the screenshot above a minimal reproduction of your issue? If not, could you share a repo that reproduces this issue?
1. Could you also share your `schema.prisma`?

**Reply by apprat** [reply]
> Hey @apprat 👋 ,
> 
> I'm unsure what problem you are running into.
> 
> 1. Would you mind sharing what results you are expecting vs. what is actually happening?
> 2. Is the screenshot above a minimal reproduction of your issue? If not, could you share a repo that reproduces this issue?
> 3. Could you also share your `schema.prisma`?

I enabled transaction (begin) in route A. Since prisma shares the same client, route B will also run in begin.

**Reply by janpio** [reply]
You do not get any guarantee that the next query will use the same connection unless you specifically request that with special syntax: https://www.prisma.io/docs/concepts/components/prisma-client/transactions

**Reply by vladi-stevanovic** [reply]
👋 Hello @apprat I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
